File: beecrowd/1/1020/1020.go
package main

import (
	"fmt"
)

func main() {
	var dias int
	var tempo = [3]int{365, 30, 1}
	fmt.Scan(&dias)
	for i, v := range tempo {
		tempo[i] = dias / v
		dias -= tempo[i] * v
	}
	fmt.Printf("%d ano(s)\n%d mes(es)\n%d dia(s)\n", tempo[0], tempo[1], tempo[2])
}

File: beecrowd/1/1020/1020.clj
(defn day-conv [days]
  (do
    (def y (quot days 365))
    (def m (quot (- days (* y 365)) 30))
    (def d (- days (* y 365) (* m 30)))
    (printf
      "%d ano(s)\n%d mes(es)\n%d dia(s)\n"
      y
      m
      d)))

(day-conv (read))
